specs/provider.yaml

```diff
@@ -911,7 +946,7 @@ paths:
         pattern: ^[0-9abcdefghjkmnpqrtuvwxyz]{29}$
         format: base32ID
       tags:
-      - Single Sign-On (SSO)
```

Comment:
I'd keep this one, some people don't know SSO stands for Single Sign-on (I certainly didn't :P )

Reply:
Waits that's the tag, nevermind, carry one